src/mscorlib/shared/System/Void.cs

```diff
@@ -10,7 +10,7 @@
 namespace System
 {
     // This class represents the void return type
-    public struct Void
+    public readonly struct Void
```

Comment:
You can mark this readonly, but it has zero value here - just adds extra bytes to the binary. The marking on the type looks a bit odd to me.

I am wondering what the rule should be: mark structs as read only because of it make sense (it would be my preference), or just because of we can?

Reply:
Yeah, I went back and forth on that.  There are a few types like this in a PR I'm about to put up in corefx as well.  I could go either way.

Reply:
Perhaps mark like `public /* readonly */ struct Void` so it's clear it was intentionally skipped.

Reply:
> Perhaps mark like public /* readonly */ struct Void so it's clear it was intentionally skipped.

Ok, I'll do that.